src/Intro/Intro.jsx

```diff
@@ -47,12 +47,16 @@ const Intro = props => (
 	<div>
 		<section id="intro" className={props.classes.section}>
 			<Hidden smUp>
-				<ResponsiveImage path="/images/skystars.jpg" asBackgroundImage />
+				<ResponsiveImage
+					className={props.classes.fullVideo}
```

Comment:
Probably related to [the removal of `"react-scripts": "2.0.0-next.b2fd8db8"`](#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L21)

Css modules support was introduce in v2: https://github.com/facebook/create-react-app/pull/2285#issuecomment-358532933

Reply:
I don't understand, should I do something?

Reply:
What I was trying to say is that the component `ResponsiveImage` was looking good, it has the [proper styles to look *fullscreen*](https://github.com/code-star/codestar-website-react/blob/develop/src/ResponsiveImage/ResponsiveImage.module.css) 
```
import css from './ResponsiveImage.module.css';
```

```
className={`${props.asBackgroundImage ? css.asBackgroundImage : ''} 
  props.className
}`}
```

```
.asBackgroundImage {
  display: block;
  position: absolute;
  min-width: 100%;
  min-height: 100%;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  z-index: -1;
}
```

Those styles are not being applied / imported.

Why is no longer working? I think it is because react supports `xxx.module.css` imports only from the package >= `react-scripts@2.0` and [it was removed](https://github.com/code-star/codestar-website-react/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L21) when you added `custom-react-scripts`, [which internally use `react-scripts@1.1.14`](https://github.com/kitze/custom-react-scripts#-custom-react-scripts-)

Reply:
Enable css modules in custom-react-scripts see https://www.npmjs.com/package/custom-react-scripts